qCC/ccColorFromScalarDlg.cpp

```diff
@@ -44,12 +44,21 @@ ccColorFromScalarDlg::ccColorFromScalarDlg(QWidget* parent, ccPointCloud* pointC
 {
 	m_ui->setupUi(this);
 	
+	
+	if (m_cloud->getCurrentDisplayedScalarFieldIndex() == -1)
```

Comment:
Should we test that there are any scalar field at all first? like cloud->hasScalarFields() or getScalarFieldCount() (can't remember)

Reply:
In theory (at least) the tool should not be able to be launched unless the cloud has at least one scalar field. Regardless, it can't hurt to check I suppose :)